.github/workflows/master.yml

```diff
@@ -52,27 +52,33 @@ jobs:
     - name: Install system dependencies
       run: |
         sudo apt-get install -y libcurl4-openssl-dev
-    - name: Install dependencies
+    - name: Install R dependencies
+      shell: Rscript {0}
       run: |
         install.packages("devtools")
         remotes::install_deps('mlflow/R/mlflow', dependencies = TRUE, upgrade = FALSE)
-      shell: Rscript {0}
-    - name: Run tests
+    - name: Build package
+      working-directory: mlflow/R/mlflow
       run: |
-        source ./dev/install-common-deps.sh
-        cd mlflow/R/mlflow
         R CMD build .
```

Comment:
I wonder if we can use `devtools::install(build = TRUE)` (or similar) to simplify `R CMD build .` and 
```r
parent_dir <- dir("../", full.names = TRUE)
 package <- parent_dir[grepl("mlflow_", parent_dir)]
 install.packages(package)
```